Example/PrebidDemo/PrebidDemoSwift/BannerController.swift

```diff
@@ -53,55 +49,98 @@ class BannerController: UIViewController, GADBannerViewDelegate, MPAdViewDelegat
 
         if (adServerName == "DFP") {
             print("entered \(adServerName) loop" )
-            loadDFPBanner(bannerUnit: bannerUnit)
+            setupAndLoadAMBanner()
+//            setupAndLoadAMBannerVAST()
 
         } else if (adServerName == "MoPub") {
```

Comment:
Same ad Android, modify the UI to have the video and video interstitial demo, instead of commenting code to show the demo.